eitprocessing/eit_data/sentec.py

```diff
@@ -18,4 +34,144 @@ def _from_path(  # pylint: disable=too-many-arguments
         max_frames: int | None = None,
         return_non_eit_data: bool = False,
     ) -> Self | tuple[Self, ContinuousDataCollection, SparseDataCollection]:
-        ...
+
+        with open(path, "br") as fh:
+            # Find the length of the file
+            fh.seek(0, 2)
+            file_length = fh.tell()
+
+            # go back to the beginning of the file
+            fh.seek(0, 0)
+
+            # instantiate reader
+            reader = Reader(fh)
+
+            # read the version int8
+            version = reader.unsigned_char('little')
+
+            if version < 2:
+                warnings.warn(f'File version {version}. Version 2 or higher expected.')
```

Comment:
Why is this a warning and not an exception?
Was this code tested with any version higher than 2? If not, shouldn't it just
check for version 2 and nothing else?